static/sass/custom/_search-results.scss

```diff
@@ -89,6 +89,33 @@
       margin-top: -0.5rem;
     }
   }
+
+  .p-contextual-menu {
+    &__toggle {
+      text-transform: capitalize;
+      
+      &--active {
+        border-color: $color-focus;
+      }
+    }
+
+    &__dropdown {
+      min-width: 19rem;
+      padding: 0.5rem 0;
+
+      .p-list__item {
+        padding: 0.5rem 1rem;
+      }
+
+      .series-tag:last-child {
+        margin-right: 0;
+      }
+
+      .series-tag--active {
+        background-color: $color-light;
```

Comment:
Doesn't look like your using this. Can it be removed?

Reply:
I've moved this to be for `p-list__item--active` as that makes more sense. It is, but I need some clarification from Spencer.